planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -1518,7 +1533,10 @@ void ObstacleStopPlannerNode::resetExternalVelocityLimit(const double current_ac
 void ObstacleStopPlannerNode::publishDebugData(
   const PlannerData & planner_data, const double current_acc)
 {
+  mutex_.lock();
   const auto & current_vel = current_velocity_ptr_->twist.twist.linear.x;
```

Comment:
```suggestion
  const auto current_vel = current_velocity_ptr_->twist.twist.linear.x;
```
same comment
https://github.com/autowarefoundation/autoware.universe/pull/1330/files#r919644038

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1330/commits/fc819f857be1b2e812d5a9b062e9e8ba1c7b9bd4